### DISCUSSION TITLE
Would it be helpful to have advanced search such as regex based search box on Task Log page

### DISCUSSION THREAD

**Discussion by Dinghang** [apache/airflow#36544] [Ideas] [answered]
People usually search keyword directly in the browsers to find certain logs on Task Log page. Wondering if it would be good in OS airflow to have regex based search box or integrate it with the log level dropdown to have some sort advance search options to make it easier for people to find and highlight certain logs

Pinterest customized airflow version already has this small feature for our users. Wondering if this can be also useful for OS as well.

**Comment by potiuk** [comment, marked as answer]
Yeah. I think it's a good idea, depending on the implementation details - but any improvements in the UX/Usabiity if it does not clutter the UI is a bonus.
